### PR TITLE
New design

### PR DESCRIPTION
**Pull request by CaetanoJS**
New design's emails and changes in some Rspec to test correctly

### CODE REVIEW

File: app/assets/stylesheets/app/mailers/layout.scss [hunk 1]
```diff
@@ -4,6 +4,89 @@
...
+    margin-bottom: 1.5em;
+    img {
+      width: 100%;
+      height: auto;
+    }
+  }
+  .header_message {
+    color: $m-mail-header-footer-color;
+    text-align: center;
+    font-size: 22px;
```
Comment (RafaFP) [inline app/assets/stylesheets/app/mailers/layout.scss]: Please use definitions for font-sizes (for example "$m-font-fields: 22px;"), you could create a definition specific for e-mail font-sizes.

File: app/assets/stylesheets/app/mailers/layout.scss [hunk 2]
```diff
@@ -4,6 +4,89 @@
...
+    image_icon {
+      margin: 0 10px;
+    }
+  }
+  .footer {
+    margin: 0 auto;
+    width: 30%;
+    text-align: center;
+    label{
+      font-size: 12px;
```
Comment (RafaFP) [inline app/assets/stylesheets/app/mailers/layout.scss]: Please use definitions for font-sizes

File: app/assets/stylesheets/app/mailers/layout.scss [hunk 3]
```diff
@@ -4,6 +4,89 @@
...
+      color: $m-grey-email-color;
+      display: block;
+    }
+    .cancel {
+      color: $m-mail-header-footer-color;
+      font-size: 10px;
```
Comment (RafaFP) [inline app/assets/stylesheets/app/mailers/layout.scss]: Please use definitions for font-sizes

File: app/assets/stylesheets/app/mailers/layout.scss [hunk 4]
```diff
@@ -4,6 +4,89 @@
...
+    }
+  }
+  .footer_message {
+    display: block;
+    color: $m-grey-email-color;
+    text-align: center;
+    line-height: 1.5em;
+    font-size: 22px;
```
Comment (RafaFP) [inline app/assets/stylesheets/app/mailers/layout.scss]: Please use definitions for font-sizes

File: app/assets/stylesheets/app/mailers/layout.scss [hunk 5]
```diff
@@ -4,6 +4,89 @@
...
+    margin-top: 30px;
+    .contact {
+      color: $m-mail-header-footer-color;
+      font-weight: bold;
+      font-size: 15px;
```
Comment (RafaFP) [inline app/assets/stylesheets/app/mailers/layout.scss]: Please use definitions for font-sizes

File: app/assets/stylesheets/app/mailers/layout.scss [hunk 6]
```diff
@@ -4,6 +4,89 @@
...
+  .footer_message {
+    display: block;
+    color: $m-grey-email-color;
+    text-align: center;
+    line-height: 1.5em;
+    font-size: 22px;
+    margin-top: 30px;
+    .contact {
+      color: $m-mail-header-footer-color;
+      font-weight: bold;
```
Comment (RafaFP) [inline app/assets/stylesheets/app/mailers/layout.scss]: Please use definitions for font-weight (**$m-font-weight-bold**)

File: app/assets/stylesheets/app/mailers/layout.scss [hunk 7]
```diff
@@ -4,6 +4,89 @@
...
+  .button {
+    a {
+      color: $m-mail-link-color;
+      cursor: pointer;
+      background-color: $m-mail-header-footer-color;
+      padding: 15px 32px;
+      text-align: center;
+      text-decoration: none;
+      display: block;
+      font-size: 20px;
```
Comment (RafaFP) [inline app/assets/stylesheets/app/mailers/layout.scss]: Please use definitions for font-sizes

File: app/assets/stylesheets/_definitions.scss [hunk 8]
```diff
@@ -43,6 +43,7 @@ $m-success-notification: #7AB29B;
 $m-success-notification-border: 5px solid #396554;
 $m-error-notification: #D5455A;
 $m-error-notification-border: 5px solid #6A232D;
+$m-grey-email-color: #848D95;
```
Comment (RafaFP) [inline app/assets/stylesheets/_definitions.scss]: "grey" is gramatically correct, but all other similar colors use "gray", please follow the pattern

File: app/assets/stylesheets/app/mailers/web_conference_mailer.scss [hunk 9]
```diff
@@ -8,7 +8,7 @@
     margin: 3px 0;
 
     label {
-      color: $m-dark-gray;
+      color: $m-grey-email-color;
       display: inline;
       font-weight: bold;
```
Comment (RafaFP) [inline app/assets/stylesheets/app/mailers/web_conference_mailer.scss]: Please use definitions for font-weight

File: app/mailers/views/application_mailer/feedback_email.html.haml [hunk 10]
```diff
@@ -1,6 +1,7 @@
+.header_image
+  =image_tag URI.join(root_url, "assets/mailer/feedback.png"), size: '494x263'
 = t('.content', email: @email).html_safe
 %br
 %br
-
-= @text
-%br
+.feedback_text
+  %q= @text
```
Comment (RafaFP) [inline app/mailers/views/application_mailer/feedback_email.html.haml]: Is this text a quotation? Why is it tagged <q>?

Comment (RafaFP) [inline app/mailers/views/application_mailer/feedback_email.html.haml]: Commented before, is this text a quotation?

File: app/views/layouts/mailers.html.haml [hunk 11]
```diff
@@ -5,9 +5,19 @@
     = stylesheet_link_tag "mailers", :media => "screen, projection"
     = yield :headers
   %body{:class => "#{@mailer_name} #{@mail_name}"}
+    .header-logo
+      =   image_tag URI.join(root_url, "assets/mailer/mconf_live.png"), size: '277x37'
```
Comment (RafaFP) [inline app/views/layouts/mailers.html.haml]: There is a lot of space here

File: spec/mailers/admin_mailer_spec.rb [hunk 12]
```diff
@@ -81,6 +82,10 @@
       it("assigns @user_name") {
         mail_content(mail).should match(user.name)
       }
+      #it("assigns user.name in the message") {
```
Comment (RafaFP) [inline spec/mailers/admin_mailer_spec.rb]: Code that will no longer be used can be removed

File: app/assets/stylesheets/app/mailers/layout.scss (hunk 1 above)
Comment (CaetanoJS) [reply app/assets/stylesheets/app/mailers/layout.scss]: changed to: $m-mail-header-footer-font-size

File: app/assets/stylesheets/app/mailers/layout.scss (hunk 2 above)
Comment (CaetanoJS) [reply app/assets/stylesheets/app/mailers/layout.scss]: changed to: $m-mail-footer-label-font-size

File: app/assets/stylesheets/app/mailers/layout.scss (hunk 3 above)
Comment (CaetanoJS) [reply app/assets/stylesheets/app/mailers/layout.scss]: changed to: $m-mail-footer-cancel-font-size

File: app/assets/stylesheets/app/mailers/layout.scss (hunk 4 above)
Comment (CaetanoJS) [reply app/assets/stylesheets/app/mailers/layout.scss]: changed to: $m-mail-header-footer-font-size

File: app/assets/stylesheets/app/mailers/layout.scss (hunk 6 above)
Comment (CaetanoJS) [reply app/assets/stylesheets/app/mailers/layout.scss]: changed to: $m-font-weight-bold

File: app/assets/stylesheets/_definitions.scss (hunk 8 above)
Comment (CaetanoJS) [reply app/assets/stylesheets/_definitions.scss]: changed to : "gray"

File: app/mailers/views/application_mailer/feedback_email.html.haml (hunk 10 above)
Comment (CaetanoJS) [reply app/mailers/views/application_mailer/feedback_email.html.haml]: Because i wanna let the text between quotations

Comment (CaetanoJS) [reply app/mailers/views/application_mailer/feedback_email.html.haml]: same as before

File: spec/mailers/admin_mailer_spec.rb (hunk 12 above)
Comment (CaetanoJS) [reply spec/mailers/admin_mailer_spec.rb]: already removed

File: app/views/layouts/mailers.html.haml (hunk 11 above)
Comment (CaetanoJS) [reply app/views/layouts/mailers.html.haml]: done!!

File: app/assets/stylesheets/app/mailers/layout.scss (hunk 6 above)
Comment (CaetanoJS) [reply app/assets/stylesheets/app/mailers/layout.scss]: changed to: $m-font-weight-bold

File: app/assets/stylesheets/app/mailers/web_conference_mailer.scss (hunk 9 above)
Comment (CaetanoJS) [reply app/assets/stylesheets/app/mailers/web_conference_mailer.scss]: changed to: $m-font-weight-bold

File: app/assets/stylesheets/app/mailers/layout.scss (hunk 5 above)
Comment (CaetanoJS) [reply app/assets/stylesheets/app/mailers/layout.scss]: $m-mail-footer-contact-font-size

File: app/assets/stylesheets/app/mailers/layout.scss (hunk 7 above)
Comment (CaetanoJS) [reply app/assets/stylesheets/app/mailers/layout.scss]: $m-mail-button-font-size

**Pull request merged**